.github/pull_request_template.md

```diff
@@ -1,18 +1,20 @@
+# Description
+
 <!--
 What is the purpose of this PR?
 
-* What is the context surrounding this PR? Include links if possible.
-* What kind of feedback do you want?
 * Have you [formatted the PR title](https://github.com/pinterest/gestalt/#releasing)? `ComponentName: Description`
 -->
 
-## Test Plan
+## Links
 
-<!--
-How can reviewers verify this is good to merge?
+- JIRA: https://jira.pinadmin.com/browse/PDS-XXX
+- [TDD](paper doc link)
+- [Figma](link to figma file)
 
-* Is it tested?
-* Is it accessible?
-* Is it documented?
-* Have you involved other stakeholders (such as a Pinterest Designer)?
--->
+## Test Plan
+
+- [ ] Added Unit Tests
+- [ ] Added Accessibility Tests
+- [ ] Visually checked for regressions
```

Comment:
What does this mean?

Reply:
Ended up removing it, but it was a reminder to look at related components when making a "breaking" change